src/views/Home/Home.js

```diff
@@ -1,18 +1,35 @@
-import React from 'react';
+import React, { useState } from 'react';
+import './Home.css';
 
 import Preview from '../../components/Preview/Preview';
 import Editor from '../../components/Editor/Editor';
 
-import './Home.css';
 
+// add useState calls here for title, subtitle, font, align, and text
 export default function Home() {
-  // add useState calls here for title, subtitle, font, align, and text
+
+  const [title, setTitle] = useState('');
+  const [subtitle, setSubtitle] = useState('');
+  const [titleFont, setTitleFont] = useState('');
+  const [align, setAlign] = useState('');
```

Comment:
If we call useState('center') for our titleFont function, this will allow the program to default to center :)